Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/Implementation/Experimental/ExperimentalFeaturesTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Experimental
+{
+    [TestClass]
+    public class ExperimentalFeaturesTests
+    {
+        [TestMethod]
+        public void VerifyExperimentalFeaturesExtensionIfConfigEmpty()
+        {
+            var config = new TelemetryConfiguration();
+
+            bool value = config.EvaluateExperimentalFeature("abc");
+            Assert.IsFalse(value);
+        }
+
+        [TestMethod]
+        public void VerifyExperimentalFeaturesExtension()
+        {
+            var config = new TelemetryConfiguration
+            {
+                ExperimentalFeatures = new string[] { "abc" }
+            };
+
+            bool value = config.EvaluateExperimentalFeature("abc");
+            Assert.IsTrue(value);
+
+            bool fakeValue = config.EvaluateExperimentalFeature("fake");
+            Assert.IsFalse(fakeValue);
+        }
```

Comment:
add a case where someone is adding a experimental feature. I dont see an API to add a feature?